conda-forge.yml

```diff
@@ -9,6 +9,5 @@ github:
   tooling_branch_name: main
 os_version:
   linux_64: cos7
-provider:
-  linux_aarch64: default
-test_on_native_only: true
+# Emulated tests exceed the 6h CI threshold
+test: native
```

Comment:
I would prefer just skipping the test suite invocation on aarch, so we at least still have the import tests.

Reply:
Adjusted.